### ISSUE TITLE
RFC: config on object key string unquote

### ISSUE THREAD

**Issue by 94noni** [VincentLanglet/Twig-CS-Fixer#312]
### Rule(s) related to or rule(s) missing

Perhaps related to [HashQuoteRule](https://github.com/VincentLanglet/Twig-CS-Fixer/blob/main/src/Rules/String/HashQuoteRule.php)

### Expected behavior

asking if we can have a config where all the changes of the below image are done EXCEPT the part `'user': user`

### Actual behavior

<img width="844" alt="twigcs" src="https://github.com/user-attachments/assets/d8d93c47-372e-453a-bec2-011f75cd222c">

**Comment by 94noni** [comment]
I've quite tried it but had error like 

ERROR 43 | {{ include('@template/.../..., {
44 | 'user': {
>> | The hash key "user" does not require to be quoted.


things like that

**Comment by 94noni** [comment]
I've tried this 

```php
<?php

use TwigCsFixer\Rules\String\HashQuoteRule;

$config = new TwigCsFixer\Config\Config();

$ruleset = new TwigCsFixer\Ruleset\Ruleset();

$ruleset->addStandard(new TwigCsFixer\Standard\TwigCsFixer());

$ruleset->addRule(new HashQuoteRule(useQuote: true));
// without, the tool swap the object keys quote
// with, the tool output some errors

$config->setRuleset($ruleset);

return $config;

```

**Comment by VincentLanglet** [comment]
You're using the wrong method
- Add is to add a new rule
- Override is to override an already added rule

```
$ruleset->overrideRule(new HashQuoteRule(useQuote: true));
```

**Comment by 94noni** [comment]
perhaps I've misread the doc :( 
I feel very sorry for having taken your time for such useless things

it works perfectly ! thank a lot @VincentLanglet

**Comment by VincentLanglet** [comment]
cf https://github.com/VincentLanglet/Twig-CS-Fixer/blob/main/docs/configuration.md

<img width="735" alt="image" src="https://github.com/user-attachments/assets/bc990e7f-95c1-45c6-924f-15246d69329e">

Maybe it's not clear enough, PR is welcomed.

**Comment by 94noni** [comment]
yes I think I've misunderstood the fact that this rule is enabled in the default fixer, thus the method to call was override and not add

just reasoning out loud, is it doable that the config knows that the rule is by default enabled, thus calling addRule internally fallback to overriderule ?
